### PR TITLE
Add link to derivation docs

### PR DESCRIPTION
**Pull request by lserra-iov**
Add a link to the derivation docs in the transaction summary when a peg-out is finished.

### CODE REVIEW

**Review by alexjavabraz — APPROVED** [review]
LGTM

File: src/common/components/exchange/TxSummaryFixed.vue [hunk 1]
```diff
@@ -300,6 +300,15 @@
             </v-col>
 
           </v-row>
+          <v-row v-if="releasedPegout" class="mx-1 mt-6">
+            <span class="next-steps">
+              Next steps: Click
+              <a :href="appConstants.DERIVE_BTC_ADDRESS_DOCUMENTATION_URL"
+                class="d-inline text-decoration-underline font-weight-bold"
+                target="_blank">here</a>
```
Comment (ronaldsg20) [inline src/common/components/exchange/TxSummaryFixed.vue:308]: I think you can add `rel="noopener"` to de `a` tag

**Review by ronaldsg20 — APPROVED** [review]
LGTM

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Pull request merged**